### DISCUSSION TITLE
Fixing Errors on MacOS (M1, MacOS Sonoma)

### DISCUSSION THREAD

**Discussion by SidSheth007** [BindsNET/bindsnet#682] [General]
Hi all,

Just wanted to share a quick fix I found for some errors when attempting to run eth_mnist.py and other example tests on MacOS with homebrew and pyenv.

If receiving an error report indicating that an _lzma module is missing, use this enable framework flag...

PYTHON_CONFIGURE_OPTS="--enable-framework" pyenv install 3.9.6

This fixed any issues for me!

[https://stackoverflow.com/questions/59690698/modulenotfounderror-no-module-named-lzma-when-building-python-using-pyenv-on](url)

**Comment by Hananel-Hazan** [comment]
Thank you for the info!
